METIS/docs/example_notebooks/LSS-YSO_model_simulation.ipynb

```diff
@@ -611,8 +619,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "std_result[1].data -= bgresult[1].data / exptime\n",
-    "std_rectified = tracelist.rectify_traces(std_result, -4, 4)"
+    "std_result[1].data -= bgresult[1].data / exptime * std_exptime\n",
```

Comment:
Since `std_exptime = 1 s`, this shouldn't make a difference, but of course it's cleaner to write down the full Dreisatz.

Reply:
Indeed I noticed that this is just a multiplication by one. However, like you mention, I think this way it's a lot clearer what's happening physically. Also, with all-consistent unit, we would end up with a dimensional problem otherwise 😅